core/src/main/java/com/onewelcome/core/util/TestConstants.kt

```diff
@@ -65,4 +69,22 @@ object TestConstants {
       //no-op
     }
   }
+
+  class FakePinAuthenticationRequestHandler() : OneginiPinAuthenticationRequestHandler {
+    override fun startAuthentication(
+      userProfile: UserProfile,
+      callback: OneginiPinCallback,
+      attemptCounter: AuthenticationAttemptCounter
+    ) {
+      //no-op
+    }
+
+    override fun onNextAuthenticationAttempt(attemptCounter: AuthenticationAttemptCounter) {
+      //no-op
+    }
+
+    override fun finishAuthentication() {
+      //no-op
+    }
+  }
```

Comment:
This one is not used. But maybe we could move those fake classes to the tests package `app/src/test/java/com/onewelcome/showcaseapp/fakes` ? 🤔